### PR TITLE
Switch to html-minifier-terser to handle ES6 minification correctly

### PR DESCRIPTION
**Pull request by kbiedrzycki**
We tried to implement ember-cli-html-minifier and it worked fine to some degree, however, we found out that if we add some content to HTML file that is a javascript code written using ES6 (or higher) it won't be handled correctly by uglifyJS used by `html-minifier`. This change switches minification base to `html-minifier-terser` which solves the issue.

### CODE REVIEW

**Comment by kbiedrzycki** [comment]
Hey @gpoitch, feel free to go trough the change and let me know if it works for you.

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "minify",
     "compress",
     "html",
-    "html-minifier"
+    "html-minifier-terser"
```
Comment (gpoitch) [inline package.json]: Don't need this keyword change

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,455 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (gpoitch) [inline yarn.lock]: Can remove this file

**Review by gpoitch — COMMENTED** [review]
Thanks @kbiedrzycki

**Comment by kbiedrzycki** [comment]
@gpoitch done!

**Pull request merged**